apps/extension/server/src/server.ts

```diff
@@ -216,6 +217,11 @@ async function handleDocumentChange(textDocument: TextDocument): Promise<void> {
 
     if (displayedMachine?.uri === textDocument.uri) {
       const machineResult = machineResults[displayedMachine.machineIndex]!;
+      // If we got this far we can safely assume that the machine config is valid and we can clear any potential errors
+      connection.sendNotification('extensionError', {
+        message: undefined,
+      });
```

Comment:
This is true and it works fine but I feel like when debugging what's going on in the server-client communication this might get a little bit spammy. I'd try to only send this out if we actually have a displayed error in the first place.